src/Widget/Field.php

```diff
@@ -410,7 +411,7 @@ public function passwordInput(array $options = []): self
      *
      * @return self the field object itself.
      */
-    public function fileInput(array $options = []): self
+    public function fileInput(bool $noHiddenInput = false, array $options = []): self
```

Comment:
```suggestion
    public function fileInput(bool $withoutHiddenInput = false, array $options = []): self
```

Also it may make sense to swap arguments order:

```suggestion
    public function fileInput(array $options = [], bool $withoutHiddenInput = false): self
```

Reply:
Done.